projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -373,7 +381,9 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
     /** @hidden @internal */
     public handleOpened(): void {
         this.triggerCheck();
+        if (!this.comboInput.focused) {
         this.dropdownContainer.nativeElement.focus();
```

Comment:
```suggestion
            this.dropdownContainer.nativeElement.focus();
```

Reply:
We may also want to update the changelog with the new behavior of the component.